gazebo_dev/package.xml

```diff
@@ -17,8 +18,15 @@
 
```

Comment:
Sorry, I just realized the url lines have been removed for `gazebo_msgs`, mind bringing them back?

Reply:
94aba73